### DISCUSSION TITLE
Birdseye continuous mode - skip disabled cameras

### DISCUSSION THREAD

**Discussion by tootai** [blakeblackshear/frigate#14478] [Ask A Question] [answered]
Hi
is there a way in birdseye continuous mode to skip disabled cameras. I have some configured in birdseye that are only activated when nobody is at home. I tried mode=motion with inactivity_threshold: 3600 but this doesn't do the job.

Other possibility?

Daniel

**Comment by NickM-27** [comment]
Disabled cameras should not be possible to show up since they are not having frames created

**Reply by tootai** [reply]
Cameras don't show up but their configuration, see attached print screen
![Capture d’écran du 2024-10-21 16-46-33](https://github.com/user-attachments/assets/1e50b4c3-e7d6-4918-a21f-fb83ac224804)

That's what I want to avoid

**Reply by NickM-27** [reply, marked as answer]
This is the frigate card not frigate

**Reply by tootai** [reply]
Sorry for this, thanks for pointing it.
